src/components/input/bl-input.css

```diff
@@ -115,51 +165,42 @@ label {
   padding: 0;
 }
 
-.dirty:invalid ~ label {
-  color: var(--bl-color-danger);
+::part(button) {
+  --padding-horizontal: 5px;
+  --text-hover-color: transparent;
 }
```

Comment:
Opening a `part` means this internal button element can be styles in any way outside of the component. I don't think that we want this. I see that you wanted to override hover background and paddings. But we should either not need this or should fix in another way.

I think a standart `tertiary`, `small` and `neutral` icon only button should be fine for here.

Reply:
Is there any way to override button's style? If I keep the button size to medium I have to remove the hover background, otherwise I have to set the size to small but then the icon size will be smaller compared to the next icon to it.